.env.development

```diff
@@ -0,0 +1,7 @@
+API_URL=http://host.docker.internal:3000
```

Comment:
Any service that needs to be reachable server and client side will need to use host.docker.internal:<PORT> in their env. This also requires developers to add 127.0.0.1 host.docker.internal to their /etc/hosts/ file.